docs/content/en/docs/clustermgmt/cluster-upgrades/vsphere-and-cloudstack-upgrades.md

```diff
@@ -30,6 +30,14 @@ The changes in the EKS Anywhere controller can trigger a machine rollout on the
 
 {{% /alert %}}
 
+### Prepare DHCP IP addresses pool
+
+Please make sure to have sufficient available IP addresses in your DHCP pool to cover the new machines. The number of necessary IPs can be calculated from the machine counts and [maxSurge config]({{< ref "./baremetal-upgrades.md/#upgraderolloutstrategyrollingupdatemaxsurge" >}}). For create operation, each machine needs 1 IP. For upgrade operation, control plane and workers need just 1 extra IP (total, not per node) due to rolling upgrade strategy. Each external ETCD machine needs 1 extra IP address (ex: 3 ETCD nodes would require 3 more IP addresses) because EKS Anywhere needs to create all the new ETCD machines before removing any old ones. You would also need additional IPs to be equal to the number used for maxSurge. After calculating the required IPs, please make sure your environment has enough available IPs before the upgrade operation.
```

Comment:
```suggestion
Please make sure to have sufficient available IP addresses in your DHCP pool to cover the new machines. The number of necessary IPs can be calculated from the machine counts and [maxSurge config]({{< ref "./baremetal-upgrades.md/#upgraderolloutstrategyrollingupdatemaxsurge" >}}). For create operation, each machine needs 1 IP. For upgrade operation, control plane and workers need just 1 extra IP (total, not per node) due to rolling upgrade strategy. Each external ETCD machine needs 1 extra IP address (ex: 3 ETCD nodes would require 3 more IP addresses) because EKS Anywhere needs to create all the new ETCD machines before removing any old ones. You will also need additional IPs to be equal to the number used for maxSurge. After calculating the required IPs, please make sure your environment has enough available IPs before performing the upgrade operation.
```